ondewo_bpi_qa/bpi_qa_server.py

```diff
@@ -46,19 +53,22 @@ def __init__(self) -> None:
     def serve(self) -> None:
         super().serve()
 
-    @Timer(log_arguments=False)
-    def DetectIntent(self, request: DetectIntentRequest, context: grpc.ServicerContext) -> DetectIntentResponse:
+    @Timer(log_arguments=False, logger=logger_console.debug)
+    def DetectIntent(self, request: DetectIntentRequest,
+                     context: grpc.ServicerContext) -> DetectIntentResponse:
```

Comment:
format bitte

Reply:
If fits in one row:
  def func_name(param_1: type, param_2: type) -> return_type:


If it does not fit in one row
  def func_name(
      param_1: type,
      param_2: type,
  ) -> return_type:

NOT THE SAME AS:
  def func_name(
      param_1: type,
      param_2: type,) -> return_type:

But a question is: What if all params fit in the second row ?
  def func_name(
      param_1: type, param_2: type
  ) -> return_type:

Any thoughts ?

Reply:
So you think 

`@Timer(log_arguments=False, logger=logger_console.debug)
  def DetectIntent(
            self,
            request: DetectIntentRequest,
            context: grpc.ServicerContext
    ) -> DetectIntentResponse:
        pass`

is more readable than:

`@Timer(log_arguments=False, logger=logger_console.debug)
  def DetectIntent(self,
                     request: DetectIntentRequest,
                     context: grpc.ServicerContext) -> DetectIntentResponse:`

?

Reply:
```
@Timer(log_arguments=False, logger=logger_console.debug)
def DetectIntent(self,
                 request_with_a_super_long_name: DetectIntentRequest,
                 another_parameter_with_very_long_name: DetectIntentReqest
                 context: grpc.ServicerContext) -> DetectIntentResponse:
```

What about this case ?